src/Swarm/Language/Pipeline.hs

```diff
@@ -40,12 +40,12 @@ import Witch
 --   pipeline.  Put a 'Term' in, and get one of these out.
 data ProcessedTerm
   = ProcessedTerm
+      -- | The elaborated + type-annotated term, plus types of any embedded definitions
       TModule
-      -- ^ The elaborated + type-annotated term, plus types of any embedded definitions
+      -- | Requirements of the term
```

Comment:
I think Haddock chokes on this.  But once we rebase this PR on top of #1308 it should be fine.